plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -109,19 +85,42 @@ def __init__(self, **kwargs):
         self._sequential_fails = 0
         super().__init__(**kwargs)
 
-    def config_pairs(self):  # pragma: no cover TODO?
+    def config_pairs(self):
         """Returns info about the given templater for output by the cli."""
         return [("templater", self.name), ("dbt", self.dbt_version)]
 
-    @property
-    def dbt_version(self):  # pragma: no cover
+    @cached_property
+    def _dbt_version(self) -> "VersionSpecifier":
+        """Fetches the installed dbt version.
+
+        This is cached in the raw dbt format.
+
+        NOTE: We do this only on demand to reduce the amount of loading
+        required to discover the templater.
+        """
+        from dbt.version import get_installed_version
+
+        return get_installed_version()
+
+    @cached_property
+    def dbt_version(self):
+        """Gets the dbt version."""
+        return self._dbt_version.to_version_string()
+
+    @cached_property
+    def dbt_version_tuple(self):
         """Gets the dbt version."""
-        return DBT_VERSION_STRING
+        return (int(self._dbt_version.major), int(self._dbt_version.minor))
```

Comment:
pretty sure the wrapping brackets are redundant when returning tuples

Reply:
Right you are. Will fix 👍